app/src/main/java/io/github/wulkanowy/services/sync/works/AttendanceWork.kt

```diff
@@ -0,0 +1,18 @@
+package io.github.wulkanowy.services.sync.works
+
+import io.github.wulkanowy.data.db.entities.Semester
+import io.github.wulkanowy.data.db.entities.Student
+import io.github.wulkanowy.data.repositories.attendance.AttendanceRepository
+import io.github.wulkanowy.utils.friday
+import io.github.wulkanowy.utils.monday
+import io.reactivex.Completable
+import org.threeten.bp.LocalDate
+import javax.inject.Inject
+
+class AttendanceWork @Inject constructor(private val attendanceRepository: AttendanceRepository) : Work {
+
+    override fun create(student: Student, semester: Semester): Completable {
+        return attendanceRepository.getAttendance(semester, LocalDate.now().monday, LocalDate.now().friday, true)
```

Comment:
Użyj statycznego importu now() z LocalDate:

```suggestion
        return attendanceRepository.getAttendance(semester, now().monday, now().friday, true)
```